stories/chats/src/UIChatInput/styles.tsx

```diff
@@ -18,4 +18,8 @@ export const commonStyles = StyleSheet.create({
         height: UIConstant.mediumIconSize(),
         width: UIConstant.mediumIconSize(),
     },
+    actionTitle: {
+        minWidth: UIConstant.iconSize(),
```

Comment:
What's the reason to set minWidth? can you a comment here to not forget what it's for?

Reply:
I'll make an update and remove it, it won't be needed anymore))